package.json

```diff
@@ -45,8 +43,9 @@
     ]
   },
   "dependencies": {
-    "@dimensiondev/stego-js": "^0.11.0",
-    "@holoflows/kit": "https://github.com/DimensionDev/holoflows-kit",
+    "@dimensiondev/matrix-js-sdk-type": "unstable",
```

Comment:
Does this repo up to date with the upstream? (https://github.com/Jack-Works/matrix-js-sdk-type)

Reply:
synced, https://github.com/DimensionDev/matrix-js-sdk-type